features/nanostack/nanostack-interface/Nanostack.h

```diff
@@ -41,9 +41,10 @@ class Nanostack : public OnboardNetworkStack, private mbed::NonCopyable<Nanostac
 
     /* Implement OnboardNetworkStack method */
     virtual nsapi_error_t add_ethernet_interface(EMAC &emac, bool default_if, OnboardNetworkStack::Interface **interface_out);
+    virtual nsapi_error_t add_ethernet_interface(EMAC &emac, bool default_if, OnboardNetworkStack::Interface **interface_out, const uint8_t *mac_addr);
 
-    /* Local variant with stronger typing and manual address specification */
-    nsapi_error_t add_ethernet_interface(EMAC &emac, bool default_if, Nanostack::EthernetInterface **interface_out, const uint8_t *mac_addr = NULL);
+    /* Local variant */
+    nsapi_error_t add_ethernet_interface_ns(EMAC &emac, bool default_if, Nanostack::EthernetInterface **interface_out, const uint8_t *mac_addr = NULL);
```

Comment:
is this needed?

Reply:
Not needed, will be removed in the next commit.